src/libs/actions/User.js

```diff
@@ -187,6 +315,62 @@ function validateLogin(accountID, validateCode) {
     Navigation.navigate(ROUTES.HOME);
 }
 
+/**
+ * Validates a secondary login / contact method
+ *
+ * @param {String} contactMethod - The contact method the user is trying to verify
+ * @param {String} validateCode
+ */
+function validateSecondaryLogin(contactMethod, validateCode) {
```

Comment:
Looks like the `validateSecondaryLogin` was missing the `successData` data for the case where, when we have a non-validated account, add a secondary login, and then verify the secondary login, the BE actually sets the secondary login as the primary login.

It can be verified from an email received after verifying, telling that the primary login has been changed to the secondary login. So, when we try to set the secondary login as the primary login, the BE responds with a message that the secondary login is already the primary login.

Leading to this issue:
- https://github.com/Expensify/App/issues/46350

which was fixed by PR:
- https://github.com/Expensify/App/pull/47132

more details on the fix approach can be found in the [proposal](https://github.com/Expensify/App/issues/46350#issuecomment-2253721504).